src/resources/Witchess.ts

```diff
@@ -1,8 +1,36 @@
+import { myHash, runChoice, runCombat, toInt, visitUrl } from "kolmafia";
 import { haveInCampground } from "../lib";
+import { get } from "../property";
 import { $item } from "../template-string";
 
-export const item = $item`Witchess Set`;
-
 export function have(): boolean {
-  return haveInCampground(item);
+  return haveInCampground($item`Witchess Set`);
+}
+
+export function fightsDone(): number {
+  return get("_witchessFights");
+}
+
+export function fightPiece(piece: Monster): string {
+  if (
```

Comment:
Would be cool to add a check that its a valid witchess monster

Reply:
done